src/lib/kdump/kdump_calibrator.rb

```diff
@@ -8,13 +8,15 @@ class KdumpCalibrator
 
     LOW_MEM = 896
     MIN_LOW_DEFAULT = 72
-    MB_SIZE = 1048576
+    MiB_SIZE = 1048576
```

Comment:
NP: personally I'd prefer expressions like`2**20` (or in some cases hex literals), they are less prone to typos...